packages/babel-plugin-proposal-class-properties/test/fixtures/public-loose/foobar/options.json

```diff
@@ -1,7 +1,7 @@
 {
   "plugins": ["external-helpers", ["proposal-class-properties", {"loose": true}]],
   "presets": [
-    ["stage-0", { "decoratorsLegacy": true }],
+    ["stage-0", { "decoratorsLegacy": true, "pipelineProposal": "minimal" }],
```

Comment:
Unrelated to the PR but I wonder why the stage-0 preset is even necessary to use in this test?

Reply:
I have no idea, but it did catch that I broke state-0 so maybe that's ok?

Reply:
Yeah, it is ok. I often use `stage-0` in tests for lazyness 😛